pkgs/build-support/fetchgithub/default.nix

```
@@ -21,6 +24,7 @@ lib.makeOverridable (
     sparseCheckout ? [ ],
     githubBase ? "github.com",
     varPrefix ? null,
+    archiveReproducibilityWorkaround ? false,
```

I don't think we should call this a workaround. It is a reimplementation of export-subst, and ideally that would have been something that's exposed by git. Other than that, this is a good solution to the problem, even if the constraints aren't very fun.

```suggestion
    pureExportSubst ? false,
```